ios/PurchasesHybridCommon/PurchasesHybridCommon/ErrorContainer.swift

```diff
@@ -27,9 +27,11 @@ import Purchases
 
         info["underlyingErrorMessage"] = underlyingErrorMessage ?? ""
 
-        if let readableErrorCode = nsError.userInfo[Purchases.ReadableErrorCodeKey] {
+        // todo: remove "readable_error_code" and instead send whole user info instead
+        // also: code name is already exposed as error.code
```

Comment:
I am all down to rethink the errors. We just need to be careful with the JS based hybrids because we can't easily deprecate anything and we can break their code